app/com/linkedin/drelephant/spark/fetchers/statusapiv1/statusapiv1.scala

```diff
@@ -87,6 +87,7 @@ trait ExecutorSummary{
   def totalShuffleRead: Long
   def totalShuffleWrite: Long
   def maxMemory: Long
+  def totalMemoryBytesSpilled: Long
```

Comment:
Does the regular Spark history server REST API provide totalMemoryBytesSpilled? Or you are using a custom Spark history server?

Reply:
Using the custom Spark History Server